src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -390,6 +391,7 @@ private void GoToRootDirectory()
             if (Directory.Exists(Model.RootDirectory))
             {
                 Process.Start(Model.RootDirectory);
+                Analytics.TrackEvent(Actions.Open, Categories.PackageManagerOperations, $"Show Root Directory of the {Model.Name} Package");
```

Comment:
Just tracking package name should be fine, thanks!

Reply:
Done